mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+python_version = 3.7
```

Comment:
we should use something more recent, probably 3.10

Reply:
I think this needs to be the oldest supported version otherwise mypy can tell you to do things which don't work with 3.7.